tools/ansible/tasks/wsu/main.yaml

```diff
@@ -73,16 +75,26 @@
         dest: "{{ win_temp_dir.path }}\\worker.ign"
         validate_certs: no
 
+    - name: Get cluster version
+      delegate_to: localhost
+      shell: "oc get clusterversion -o json | jq -r .items[0].status.desired.version | cut -c1-3"
+      register: cluster_version
+
     - name: Get hybrid overlay exe
       win_get_url:
-        url: "https://github.com/openshift/windows-machine-config-operator/releases/download/v0.3-alpha/hybrid-overlay.exe"
+        url: "curl -s https://api.github.com/repos/openshift/windows-machine-config-operator/releases | jq '.[] | select (.tag_name == '$(curl -s https://api.github.com/repos/openshift/windows-machine-config-operator/releases |  jq '.[]' | jq '.tag_name' | grep -i {{ cluster_version }} | sort -r | awk 'NR==1')' )' | jq ' .assets[] | select (.name == \"{{ wmcb_exe }}\" )'| jq .browser_download_url"
```

Comment:
To get the tag_name, I had to query the api again, even though there are group by constructs in jq, the data model from github or the fields we've chosen is making this complicated. Not sure, if there are easier ways to do this.

Reply:
Understood. Please add a comment explaining what is going on.

Reply:
Sure.